src/libopensc/card-isoApplet.c

```diff
@@ -67,6 +71,23 @@ static struct sc_card_driver isoApplet_drv =
 	NULL, 0, NULL
 };
 
+static struct isoapplet_supported_ec_curves {
+		struct sc_object_id oid;
+		size_t size;
+		unsigned int min_applet_version;
+} ec_curves[] = {
+	{{{1, 2, 840, 10045, 3, 1, 1, -1}},     192, 0x0000}, /* secp192r1, nistp192, prime192v1, ansiX9p192r1 */
```

Comment:
@frankmorgner 
```
you are now introducing yet an other code duplication in card-isoApplet.c and pkcs15-isoApplet.c
```
I don't see where this PR introduces "yet another code duplication". Could you point me to it?
(What I infact see, is the removal of c&p code through the introduction of the table...)
This table is used to register EC curves supported by the applet. This, unfortunately, depends on the version of the applet. Thus, I need a table that maps OIDs to the version of the applet. How can this be realised with common code?
I included the length as t's not like a NIST curve would suddenly change it. Allocating seperate structs and calling sc_pkcs15_fix_ec_parameters() on each one to obtain the length would IMHO be just unnecessary and opaque bloat.

There is no common source for explicit parameters in OpenSC and that is why pkcs15-init drivers need to define them themselves. But which format is most suitable for all future cards?
For example, how are leading zero bytes handled..like in secp256k1 coefficient A - 32 bytes zero or one byte zero? The IsoApplet and Java Card API needs 32 bytes zero.

```
Also, supporting named curves and explicit parameters at the same time is not too hard...
```
I don't get your point here, sorry.